kuma/javascript/src/active-banner.jsx

```diff
@@ -189,9 +189,28 @@ function Banner(props: BannerProps) {
     );
 }
 
+export const MDN_BROWSER_COMPAT_REPORT_ID = 'mdn_browser_compat_report';
 export const DEVELOPER_NEEDS_ID = 'developer_needs';
 export const SUBSCRIPTION_ID = 'subscription_banner';
 
+function MDNBrowserCompatReportBanner() {
+    return (
+        <Banner
+            id={MDN_BROWSER_COMPAT_REPORT_ID}
+            classname="developer-needs"
```

Comment:
??

Reply:
Also, `classname` when it should be `className`. Apparently, linting CI didn't notice :(